assets/js/apps/badges/components/HSMDataFileCountBadge.jsx

```diff
@@ -16,9 +16,12 @@ const HSMDataFileCountBadge = ({ datasetId }) => {
 
   return (
     <Fragment>
-      <Badge variant="info" title={`${onlineFilesCount} of ${totalFileCount} Files online`}>
+      <Badge
+        bg="info"
+        title={`${onlineFilesCount} of ${totalFileCount} Files online`}
+      >
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933941)
```suggestion
            >
```